src/components/TradesWidget/TradeRow.tsx

```diff
@@ -238,7 +238,7 @@ export const TradeRow: React.FC<TradeRowProps> = (params) => {
       >
         <SplitHeaderTitle>
           <div>
-            {formatPrice(marketFillPrice)} {quoteTokenLabel}
+            {formatPrice({ price: marketFillPrice, decimals: DEFAULT_DECIMAL_PLACES })} {quoteTokenLabel}
```

Comment:
We can get by in this PR like this, but I think we should do dex-js the change and see if we can siplify this parts

Reply:
ok i;ll add issue